src/utils/errors.js

```diff
@@ -21,8 +21,16 @@ class UserNotFoundError extends AllContributorBotError {
     }
 }
 
+class ReferenceExistsError extends AllContributorBotError {
+    constructor() {
+        super(`A branch or Pull Request is already open.`)
```

Comment:
error message any suggestions?

Reply:
Perhaps `Branch ${branch} already exists`

Reply:
^ so at least it indicates which branch.